testing/trino-product-tests/src/main/java/io/trino/tests/product/BaseTestTableFormats.java

```diff
@@ -127,4 +128,39 @@ protected void testPathContainsSpecialCharacter(String schemaLocation, String pa
             onTrino().executeQuery("DROP SCHEMA %1$s.test".formatted(getCatalogName()));
         }
     }
+
+    protected void testSparkCompatibilityOnTrinoCreatedTable(String schemaLocation)
+    {
+        String baseTableName = "trino_created_table_using_parquet_" + randomNameSuffix();
+        String sparkTableName = format("%s.test_compat.%s", getSparkCatalog(), baseTableName);
+        String trinoTableName = format("%s.test_compat.%s", getCatalogName(), baseTableName);
+        try {
+            onTrino().executeQuery(format("CREATE SCHEMA %1$s.test_compat WITH (location = '%2$s')", getCatalogName(), schemaLocation));
+
+            onTrino().executeQuery(format(
+                    "CREATE TABLE %s ( " +
+                            "   a_boolean boolean, " +
+                            "   a_dummy varchar " +
```

Comment:
nit: I would rename to `a_varchar` for consistency with another column, and replace `dummy` with `test data` or something.